be/src/exprs/arithmetic_expr.cpp

```diff
@@ -189,6 +195,11 @@ class VectorizedDivArithmeticExpr final : public Expr {
 
     bool is_compilable() const override { return Type != TYPE_LARGEINT && IRHelper::support_jit(Type); }
 
+    std::string jit_func_name() const override {
```

Comment:
Have you conducted some profile test on jit_func_name(), it seems inefficient, especially for  short-term query that only return a few rows.  can we construct unique function name in a efficient way?

Reply:
we see it takes tens of us when hiting the cache. It is not a bottleneck now. For short_term query, we can close the jit according to plans cardinality later.